static/src/javascripts/projects/commercial/modules/user-features.js

```diff
@@ -23,15 +23,19 @@ const adFreeDataIsPresent = (): boolean => {
     return !isNaN(parseInt(cookieVal, 10));
 };
 
+const timeInDaysFromNow = (daysFromNow: number): string => {
```

Comment:
Well, the function returns a `getTime().toString()` and it isn't restricted to positive numbers for `daysFromNow`, although we only use positives at the moment.